content/courses/connecting-to-offchain-data/oracles.md

```diff
@@ -846,7 +846,7 @@ pub struct Deposit<'info> {
       seeds = [ESCROW_SEED, user.key().as_ref()],
       bump,
       payer = user,
-      space = std::mem::size_of::<EscrowState>() + 8
+      space =  EscrowState::INIT_SPACE + 8
```

Comment:
make a usize for ANCHOR_DISCRIMINATOR and set it to 8.

Reply:
Oh also call EscrowState just Escrow. It's a PDA, we know it stores state.

Reply:
```console
#[account]
[derive(InitSpace)]
pub struct EscrowState {
    pub unlock_price: f64,
    pub escrow_amount: u64,
}
````
 @mikemaccana  it is a struct in state.rs should i rename it

Reply:
Yes. state.rs already tells us that it's state.

Reply:
done